libdnf/dnf-reldep.c

```diff
@@ -91,10 +91,7 @@ dnf_reldep_new (DnfSack           *sack,
                 const gchar       *evr)
 {
     Pool *pool = dnf_sack_get_pool (sack);
-    Id id = pool_str2id (pool, name, 0);
-
-    if (id == STRID_NULL || id == STRID_EMPTY)
-        return NULL;
+    Id id = pool_str2id (pool, name, 1);
```

Comment:
Because if you try to create reldep for package X which is not in libsolv pool, this will return NULL. But this is totally wrong. It doesn't mean that we can't create such reldep.